### PR TITLE
[TF] GKE Cluster - fix import on some convenience fields

### PR DESCRIPTION
**Pull request by emilymye**
Fixes https://github.com/terraform-providers/terraform-provider-google/issues/3064

-----------------------------------------------------------------
# [all]
## [terraform]
container cluster - fix import on some convenience fields
### [terraform-beta]
container cluster - fix import on some convenience fields

### CODE REVIEW

File: third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 1]
```diff
@@ -577,3 +577,8 @@ $ terraform import google_container_cluster.mycluster my-gcp-project/us-east1-a/
 
 $ terraform import google_container_cluster.mycluster us-east1-a/my-cluster
 ```
+
+~> **Note:** This resource has several convenience fields that are state-only or used on creation and are not persisted by the API. If you have these fields set in config and import a cluster, you may see diffs that may or may not require actual operations against the resource. Example behavior:
```
Comment (rileykarson) [inline third_party/terraform/website/docs/r/container_cluster.html.markdown]:
```suggestion
~> **Note:** This resource has several fields that control Terraform-specific behavior and aren't present in the API. If they are set in config and you import a cluster, Terraform may need to perform an update immediately after import. Some of these updates are no-ops, and some may modify your cluster.

For example:
```

Convenience fields are fields that have been pulled up to the top level from a nested object (or, more generally, up any level but almost always to the top level). These are `"virtual"` fields that don't exist in the API; `virtual` isn't a great name user-side though. How about this message?

**Comment by emilymye** [comment]
@rileykarson I believe we already are testing the default node behavior (TestAccContainerCluster_withDefaultNodePoolRemoved for true, other tests for false). Without the new behavior, deleting the Import ignores in the tests would make them fail otherwise. Were you looking for something to test in particular?

File: third_party/terraform/resources/resource_container_cluster.go.erb [hunk 2]
```diff
@@ -2108,29 +2107,55 @@ func flattenPodSecurityPolicyConfig(c *containerBeta.PodSecurityPolicyConfig) []
 <% end -%>
 
 func resourceContainerClusterStateImporter(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```
[the remark this answers is not in the document]
Comment (emilymye) [reply third_party/terraform/resources/resource_container_cluster.go.erb]: rip indeed, very sorry

**Comment by emilymye** [comment]
=== RUN   TestAccContainerCluster_withDefaultNodePoolRemoved
=== PAUSE TestAccContainerCluster_withDefaultNodePoolRemoved
=== CONT  TestAccContainerCluster_withDefaultNodePoolRemoved
--- PASS: TestAccContainerCluster_withDefaultNodePoolRemoved (573.96s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google	574.371s

**Pull request merged**